src/server/generic_family.cc

```diff
@@ -1211,7 +1211,7 @@ void GenericFamily::Restore(CmdArgList args, ConnectionContext* cntx) {
     if (result.value()) {
       return cntx->SendOk();
     } else {
-      return cntx->SendError("Bad data formatasdfasdf");
+      return cntx->SendError("Bad data format");
```

Comment:
Since we use it a few lines below, can you kindly make this into a `const auto kBadFormatErr = "Bad data format"sv;` and use it here and below? :pray:

Reply:
Actually, this lines of code are never used:
```
case OpStatus::WRONG_TYPE:
        return cntx->SendError("Bad data format");
```
I don't see that we returning this OpStatus from `OnRestore`

Reply:
So I think it is better to just remove this lines

Reply:
sounds good to me